tests/realtikvtest/statisticstest/statistics_test.go

```diff
@@ -232,3 +236,67 @@ func checkFMSketch(tk *testkit.TestKit) {
 	tk.MustQuery(`SHOW STATS_HISTOGRAMS WHERE TABLE_NAME='employees' and partition_name="global"  and column_name="id"`).CheckAt([]int{6}, [][]any{
 		{"14"}})
 }
+
+func TestNoNeedIndexStatsLoading(t *testing.T) {
+	store, dom := realtikvtest.CreateMockStoreAndDomainAndSetup(t)
+	tk := testkit.NewTestKit(t, store)
+	tk.MustExec("use test;")
+	tk.MustExec("drop table if exists t;")
+	tk.MustExec("create table if not exists t(a int, b int, index ia(a));")
+	tk.MustExec("drop stats t;")
+	tk.MustExec("insert into t value(1,1), (2,2);")
+	require.Eventually(t, func() bool {
+		rows := tk.MustQuery("show stats_meta").Rows()
+		return len(rows) > 0
+	}, 2*time.Minute, 5*time.Millisecond)
```

Comment:
We can use this to trigger it immediately.
```suggestion
	h := dom.StatsHandle()
	require.NoError(t, h.DumpStatsDeltaToKV(true))
	require.NoError(t, h.Update(dom.InfoSchema()))
```

Reply:
Nice! Updated. I forgot we can directly call an update here.

Reply:
I tested it locally. It seems we can't trigger the bug in this way.

Reply:
Updated.